index.html

```diff
@@ -1,114 +1,125 @@
-                                    <p class="weekly-detail-title">SAT</p>
-                                </div>
-                                <div class="uk-card uk-card-default uk-card-hover uk-card-body uk-padding-remove-horizontal">
-                                    <p class="weekly-detail-card">28&deg; / 45&deg;</p>
-                                    <p class="weekly-detail-title">SUN</p>
-                                </div>
-                                <div class="uk-card uk-card-default uk-card-hover uk-card-body uk-padding-remove-horizontal">
-                                    <p class="weekly-detail-card">28&deg; / 45&deg;</p>
-                                    <p class="weekly-detail-title">MON</p>
-                                </div>
-                            </div>
-                            </div>
-                        </div>
-                    </div>
+            </div>
 
+        </div>
+    </div>
+
+    <div class="uk-card uk-card-default uk-card-body uk-width-1-2@m weather-card">
+        <div class="weekly-card-header">
+            WEEKLY FORECAST
+        </div>
+        <div class="weather-forecast-weekly">
+            <div class="uk-grid-small uk-child-width-expand" uk-grid>
+                <div class="uk-card uk-card-default uk-card-hover uk-card-body uk-padding-remove-horizontal">
+                    <p class="weekly-detail-card">28&deg; / 45&deg;</p>
+                    <p class="weekly-detail-title">WED</p>
+                </div>
+                <div class="uk-card uk-card-default uk-card-hover uk-card-body uk-padding-remove-horizontal">
+                    <p class="weekly-detail-card">28&deg; / 45&deg;</p>
+                    <p class="weekly-detail-title">THURS</p>
+                </div>
+                <div class="uk-card uk-card-default uk-card-hover uk-card-body uk-padding-remove-horizontal">
+                    <p class="weekly-detail-card">28&deg; / 45&deg;</p>
+                    <p class="weekly-detail-title">FRI</p>
+                </div>
+                <div class="uk-card uk-card-default uk-card-hover uk-card-body uk-padding-remove-horizontal">
+                    <p class="weekly-detail-card">28&deg; / 45&deg;</p>
+                    <p class="weekly-detail-title">SAT</p>
+                </div>
+                <div class="uk-card uk-card-default uk-card-hover uk-card-body uk-padding-remove-horizontal">
+                    <p class="weekly-detail-card">28&deg; / 45&deg;</p>
+                    <p class="weekly-detail-title">SUN</p>
+                </div>
+                <div class="uk-card uk-card-default uk-card-hover uk-card-body uk-padding-remove-horizontal">
+                    <p class="weekly-detail-card">28&deg; / 45&deg;</p>
+                    <p class="weekly-detail-title">MON</p>
+                </div>
             </div>
         </div>
+    </div>
+    </div>
+
+    </div>
+
```

Comment:
Not sure why this closing tag was removed, but can you re-add it? Must have gotten removed accidentally. I'll run a formatter on this file after I merge it, so that the structure is easier to read.